_data/task.yml

```
@@ -67,6 +67,16 @@ websites:
       img: rememberthemilk.png
       tfa: No
 
+    - name: Standard Notes
+      url: https://standardnotes.org/
+      twitter: standardnotes
+      img: standardnotes.png
+      email_address: help@standardnotes.org
+      tfa: Yes
+      software: Yes
+      exceptions:
+text: "Requieres standardnotes extended or self-hosted server."
```

This isn't correct YAML syntax

---

Sorry, what could I do to fix it? I copy-pastad it from steam's entry.

---

YAML is sensitive to spaces. If there's too many or not enough cases it will break the file.
Here's a working format:
```suggestion
       text: "Requieres standardnotes extended or self-hosted server."
```